api/src/main/java/org/pmiops/workbench/config/CacheConfig.java

```diff
@@ -0,0 +1,54 @@
+package org.pmiops.workbench.config;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import org.pmiops.workbench.db.dao.ConfigDao;
+import org.pmiops.workbench.db.model.Config;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@Configuration
+public class CacheConfig {
```

Comment:
Am I right in understanding that the name of this class is never references from outside code that depends on this? Calling it a `ConfigCache` or `ConfigCacheFactory` or something might be clearer for me (it's confusing to say "A CacheConfig is an object that gives you a ConfigCache"...

Reply:
The confusing thing here is there's Spring configuration, and then there's our Config / application configuration. 

Renamed to CacheSpringConfiguration. Calling it a Cache or CacheFactory is confusing since nobody actually instantiates this class.